ApollosProject/apollos-apps
# feat: Adds Postgres migrations

## DESCRIPTION

### What does this PR do, or why is it needed?

Removes the use of `sync` when running the app, which was causing issues

### How do I test this PR?

Remove any tables in your local database. Then get the [companion PR](https://github.com/ApollosProject/apollos-templates/pull/284) for templates and run apps and templates. You should get all tables with the correct structures.
